sys/net/link_layer/iQueue_MAC/beacon.c

```diff
@@ -0,0 +1,182 @@
+
+
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+#include "beacon.h"
+#include "math.h"
+#include "board.h"
+#include "debug.h"
+#define ENABLE_DEBUG  0
+
+
+node_regis_struc node_regis_list[MAX_LOCAL_NODE_NUM];
+
+
+unit32_t subframeduration=0;
+unit32_t cp_duration_us = 0;
+unit32_t tdma_duration=0;
+unit32_t sp_duration=0;
+
+
+unit8_t total_tdma_node_num =0;
+unit16_t total_tdma_slot_num =0;
```

Comment:
Hm, apparently `astyle` missed something here. Please put spaces around the `=`.